install/kubernetes/helm/istio/charts/pilot/templates/clusterrolebinding.yaml

```diff
@@ -14,6 +13,9 @@ roleRef:
   name: istio-pilot-{{ .Release.Namespace }}
 subjects:
   - kind: ServiceAccount
+{{- if .Values.global.rbacEnabled }}
     name: istio-pilot-service-account
-    namespace: {{ .Release.Namespace }}
+{{- else }}
+    name: default
```

Comment:
default value for serviceAccount is `default`, I think we do not need to add it here.

Ditto for others

Reply:
Who will use this `default` account? I saw that you are actually using `istio.default` below.

Reply:
I think you are confusing the ServiceAccount name `default` with the Secret name `istio.default`.
The default SA is generated by K8s while the secret is generated by the Citadel.